tools/sdk-cli/src/commands.rs

```diff
@@ -50,6 +50,10 @@ pub(crate) enum Commands {
         /// The external label or identifier of the payment
         #[clap(name = "label", short = 'l', long = "label")]
         label: Option<String>,
+
+        /// If use_trampoline is set, trampoline payments will be attempted.
+        #[clap(long, action)]
```

Comment:
Just to check, do you want to flip the action default now it's opt-in? `action=ArgAction::SetFalse`

Reply:
Ignore that